qa/suites/rgw/multisite-upgrade/squid-x/realm.yaml

```diff
@@ -0,0 +1,20 @@
+overrides:
+  rgw-multisite:
+    realm:
+      name: test-realm
+      is default: true
+    zonegroups:
+      - name: test-zonegroup
+        is_master: true
+        is_default: true
+        endpoints: [c1.client.0]
```

Comment:
in the rgw/multisite suite, the yamls like realms/two-zones.yaml also include this:
> enabled_features: ['resharding', 'notification_v2']

that would probably resolve this squid-x failure:
> ERROR: rgw_multi.tests.test_topic_notification_sync
An error occurred (InvalidArgument) when calling the CreateTopic operation: The 'notification_v2' zone feature must be enabled to create topics in an account